app/__init__.py

```diff
@@ -117,6 +117,11 @@ def search():
         #     user.print()
         return render_template('search.html', users=users)
 
+    @app.route('/admin_dashboard')
```

Comment:
@Johan-Cuevas May you add `@login_required` decorate like other sensitive routes?